solutions/bio1/UpsideDown/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { SCodeBlock, SText, STitle, SCode } from "components";
+import { brace } from "solutions/util";
+
+export const UpsideDown = () => {
+  return (
+    <>
+      <STitle>Idea</STitle>
+      <SText>
+        Let's first notice the large constraint on $n$. 
+        This means that we can't simply generate all numbers up to the $n-th$ number, as this would be too slow.
```

Comment:
`$n$-th` not `$n-th$`

Reply:
ofc how did u not see that that's such a stupid mistake to make I don't even believe it